2022-08-30 23:34:51 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:06 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:07 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:14 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:16 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:21 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:26 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:27 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:28 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:35:54 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:36:11 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:49:35 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:26)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 25 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 19 more
2022-08-30 23:49:47 database: wrong user or password; user: "JDBC:H2:~/IDEAPROJECTS/CONSTRUCTION_OFFICE_AUTOMATION/CONSTRUCTION_OFFICE_AUTOMATION"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:50:04 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:26)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 25 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 19 more
2022-08-30 23:51:52 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:26)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 25 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 19 more
2022-08-30 23:53:04 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:26)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 25 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 19 more
2022-08-30 23:53:41 database: wrong user or password; user: "JDBC:H2:~/IDEAPROJECTS/CONSTRUCTION_OFFICE_AUTOMATION/CONSTRUCTION_OFFICE_AUTOMATION"
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2022-08-30 23:53:53 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 13 more
2022-08-30 23:53:53 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.test(WebApp.java:972)
	at org.h2.server.web.WebApp.process(WebApp.java:241)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:152)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 13 more
2022-08-30 23:55:16 database: wrong user or password; user: ""
org.h2.message.DbException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:26)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:510)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 19 more
2022-09-02 02:52:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Order by expression "NAME" must be in the result list in this case; SQL statement:
SELECT DISTINCT  FROM LOCATION ORDER BY NAME [90068-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.query.Query.initExpression(Query.java:668)
	at org.h2.command.query.Query.initOrder(Query.java:585)
	at org.h2.command.query.Select.init(Select.java:1011)
	at org.h2.command.Parser.parseQuery(Parser.java:2570)
	at org.h2.command.Parser.parsePrepared(Parser.java:724)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 02:53:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT  * [*]DISTINCT FROM LOCATION ORDER BY NAME"; SQL statement:
SELECT  * DISTINCT FROM LOCATION ORDER BY NAME [42000-214]
2022-09-02 03:05:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT DISTINCT ID,NAME FROM LOCATION ORDER BY NAME [*]OVER()"; SQL statement:
SELECT DISTINCT ID,NAME FROM LOCATION ORDER BY NAME OVER() [42000-214]
2022-09-02 03:05:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT DISTINCT ID,NAME FROM LOCATION OVER([*]) ORDER BY NAME"; expected "identifier"; SQL statement:
SELECT DISTINCT ID,NAME FROM LOCATION OVER() ORDER BY NAME [42001-214]
2022-09-02 03:13:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE [*]!DISTINCT FROM LOCATION"; SQL statement:
DELETE !DISTINCT FROM LOCATION [42000-214]
2022-09-02 03:13:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE [*]DISTINCT FROM LOCATION"; expected "identifier"; SQL statement:
DELETE DISTINCT FROM LOCATION [42001-214]
2022-09-02 03:14:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT [*]!DISTINCT NAME FROM LOCATION"; SQL statement:
SELECT !DISTINCT NAME FROM LOCATION [42000-214]
2022-09-02 03:14:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT [*]!DISTINCT NAME FROM LOCATION"; SQL statement:
SELECT !DISTINCT NAME FROM LOCATION [42000-214]
2022-09-02 03:14:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT [*]!DISTINCT NAME FROM LOCATION"; SQL statement:
SELECT !DISTINCT NAME FROM LOCATION [42000-214]
2022-09-02 03:14:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT NOT [*]DISTINCT NAME FROM LOCATION"; expected "INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
SELECT NOT DISTINCT NAME FROM LOCATION [42001-214]
2022-09-02 03:14:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "EXIST" not found; SQL statement:
SELECT NOT EXIST NAME FROM LOCATION [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.ExpressionColumn.getNotIfPossible(ExpressionColumn.java:446)
	at org.h2.expression.condition.ConditionNot.optimize(ConditionNot.java:50)
	at org.h2.expression.Alias.optimize(Alias.java:52)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:14:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "EXIST" not found; SQL statement:
SELECT EXIST NAME FROM LOCATION [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.Alias.optimize(Alias.java:52)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:14:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT NOT [*]DISTINCT NAME FROM LOCATION"; expected "INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
SELECT NOT DISTINCT NAME FROM LOCATION [42001-214]
2022-09-02 03:14:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT NOT [*]DISTINCT NAME FROM LOCATION"; expected "INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
SELECT NOT DISTINCT NAME FROM LOCATION [42001-214]
2022-09-02 03:15:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT NOT UNIQUE [*]NAME FROM LOCATION"; expected "("; SQL statement:
SELECT NOT UNIQUE NAME FROM LOCATION [42001-214]
2022-09-02 03:15:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT UNIQUE [*]NAME FROM LOCATION"; expected "("; SQL statement:
SELECT UNIQUE NAME FROM LOCATION [42001-214]
2022-09-02 03:15:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT NOT [*]DISTINCT NAME FROM LOCATION"; expected "INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
SELECT NOT DISTINCT NAME FROM LOCATION [42001-214]
2022-09-02 03:15:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "INTERSECTS" not found; SQL statement:
SELECT INTERSECTS NAME FROM LOCATION [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.Alias.optimize(Alias.java:52)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:15:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "INTERSECTS" not found; SQL statement:
SELECT NOT INTERSECTS NAME FROM LOCATION [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.ExpressionColumn.getNotIfPossible(ExpressionColumn.java:446)
	at org.h2.expression.condition.ConditionNot.optimize(ConditionNot.java:50)
	at org.h2.expression.Alias.optimize(Alias.java:52)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:16:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "CHARACTER VARYING to BOOLEAN"; SQL statement:
SELECT NOT NAME FROM LOCATION [22018-214]
2022-09-02 03:39:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM LOCATION\000d\000a\000d\000adrop [*]table location"; SQL statement:
SELECT * FROM LOCATION

drop table location [42000-214]
2022-09-02 03:45:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "NAME" not found; SQL statement:
SELECT * FROM LOCATION WHERE NAME IN(SELECT NAME FROM LOCATION GROUP BY NAME HAVING COUNT(*) =1) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.expression.condition.PredicateWithSubquery.optimize(PredicateWithSubquery.java:37)
	at org.h2.expression.condition.ConditionInQuery.optimize(ConditionInQuery.java:160)
	at org.h2.expression.Expression.optimizeCondition(Expression.java:148)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1177)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:45:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "NAME" not found; SQL statement:
SELECT * FROM LOCATION WHERE FIRST_NAME IN(SELECT NAME FROM LOCATION GROUP BY FIRST_NAME HAVING COUNT(*) =1) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.expression.condition.PredicateWithSubquery.optimize(PredicateWithSubquery.java:37)
	at org.h2.expression.condition.ConditionInQuery.optimize(ConditionInQuery.java:160)
	at org.h2.expression.Expression.optimizeCondition(Expression.java:148)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1177)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:49:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM LOCATION \000d\000aWHERE FIRST_NAME \000d\000aIN(SELECT FIRST_NAME FROM LOCATION \000d\000aGROUP BY FIRST_NAME \000d\000aHAVING COUNT(*) =1)\000d\000a\000d\000a[*]ALTER TABLE LOCATION\000d\000aMODIFY COLUMN FIRSTNAME (name varchar(150))"; SQL statement:
SELECT * FROM LOCATION 
WHERE FIRST_NAME 
IN(SELECT FIRST_NAME FROM LOCATION 
GROUP BY FIRST_NAME 
HAVING COUNT(*) =1)

ALTER TABLE LOCATION
MODIFY COLUMN FIRSTNAME (name varchar(150)) [42000-214]
2022-09-02 03:49:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM LOCATION \000d\000aWHERE FIRST_NAME \000d\000aIN(SELECT FIRST_NAME FROM LOCATION \000d\000aGROUP BY FIRST_NAME \000d\000aHAVING COUNT(*) =1)\000d\000a\000d\000a[*]ALTER TABLE LOCATION\000d\000aMODIFY COLUMN FIRSTNAME (name varchar(150))"; SQL statement:
SELECT * FROM LOCATION 
WHERE FIRST_NAME 
IN(SELECT FIRST_NAME FROM LOCATION 
GROUP BY FIRST_NAME 
HAVING COUNT(*) =1)

ALTER TABLE LOCATION
MODIFY COLUMN FIRSTNAME (name varchar(150)) [42000-214]
2022-09-02 03:49:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM LOCATION \000d\000aWHERE FIRST_NAME \000d\000aIN(SELECT FIRST_NAME FROM LOCATION \000d\000aGROUP BY FIRST_NAME \000d\000aHAVING COUNT(*) =1)\000d\000a\000d\000a[*]ALTER TABLE LOCATION\000d\000aMODIFY COLUMN FIRSTNAME (name varchar(150))"; SQL statement:
SELECT * FROM LOCATION 
WHERE FIRST_NAME 
IN(SELECT FIRST_NAME FROM LOCATION 
GROUP BY FIRST_NAME 
HAVING COUNT(*) =1)

ALTER TABLE LOCATION
MODIFY COLUMN FIRSTNAME (name varchar(150)) [42000-214]
2022-09-02 03:52:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE LOCATION\000d\000a[*]MODIFY COLUMN FIRSTNAME (name varchar(150))"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE LOCATION
MODIFY COLUMN FIRSTNAME (name varchar(150)) [42001-214]
2022-09-02 03:52:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE LOCATION\000d\000aRENAME COLUMN FIRSTNAME [*](name varchar(150))"; expected "TO"; SQL statement:
ALTER TABLE LOCATION
RENAME COLUMN FIRSTNAME (name varchar(150)) [42001-214]
2022-09-02 03:52:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE LOCATION\000d\000aRENAME COLUMN FIRSTNAME [*]name varchar(150)"; expected "TO"; SQL statement:
ALTER TABLE LOCATION
RENAME COLUMN FIRSTNAME name varchar(150) [42001-214]
2022-09-02 03:53:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE LOCATION\000d\000aRENAME COLUMN FIRSTNAME [*]NAME varchar(150)"; expected "TO"; SQL statement:
ALTER TABLE LOCATION
RENAME COLUMN FIRSTNAME NAME varchar(150) [42001-214]
2022-09-02 03:53:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE LOCATION\000d\000aRENAME COLUMN FIRST_NAME TO NAME [*]varchar(150)"; SQL statement:
ALTER TABLE LOCATION
RENAME COLUMN FIRST_NAME TO NAME varchar(150) [42000-214]
2022-09-02 03:53:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "FIRST_NAME" not found; SQL statement:
ALTER TABLE LOCATION
RENAME COLUMN FIRST_NAME TO NAME [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:765)
	at org.h2.command.ddl.AlterTableRenameColumn.update(AlterTableRenameColumn.java:45)
	at org.h2.command.ddl.AlterTable.update(AlterTable.java:46)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:56:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "FIRST_NAME" not found; SQL statement:
SELECT FIRST_NAME FROM LOCATION 
WHERE FIRST_NAME 
IN(SELECT FIRST_NAME FROM LOCATION 
GROUP BY FIRST_NAME 
HAVING COUNT(*) =1) ORDER BY FIRST_NAME [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1170)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 03:56:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*]HAVING COUNT(*) =1) ORDER BY NAME"; SQL statement:
HAVING COUNT(*) =1) ORDER BY NAME [42000-214]
2022-09-02 03:56:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "FIRST_NAME" not found; SQL statement:
SELECT NAME FROM LOCATION 
WHERE FIRST_NAME 
IN(SELECT NAME FROM LOCATION 
GROUP BY NAME 
HAVING COUNT(*) =1) ORDER BY NAME [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.condition.ConditionInQuery.optimize(ConditionInQuery.java:161)
	at org.h2.expression.Expression.optimizeCondition(Expression.java:148)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1177)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-02 21:48:50 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
	at com.example.construction_office_automation/com.example.construction_office_automation.model.database.DatabaseConnection.dbConnect(DatabaseConnection.java:17)
	at com.example.construction_office_automation/com.example.construction_office_automation.controller.HelloController.getLocations(HelloController.java:472)
	at com.example.construction_office_automation/com.example.construction_office_automation.controller.HelloController.initialize(HelloController.java:227)
	at javafx.fxml@18.0.1/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2655)
	at javafx.fxml@18.0.1/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2548)
	at javafx.fxml@18.0.1/javafx.fxml.FXMLLoader.load(FXMLLoader.java:2516)
	at com.example.construction_office_automation/com.example.construction_office_automation.HelloApplication.start(HelloApplication.java:20)
	at javafx.graphics@18.0.1/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics@18.0.1/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics@18.0.1/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 30 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: /Users/macbook/IdeaProjects/construction_office_automation/construction_office_automation.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 24 more
2022-09-03 01:12:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE projects \000d\000a  [*]project_id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  project_name VARCHAR(100),\000d\000a  project_owner VARCHAR(50),\000d\000a  project_description VARCHAR(200),\000d\000a  status VARCHAR(20)\000d\000a  starting_date DATE,\000d\000a  finishing_date DATE,\000d\000a  date_added TIMESTAMP\000d\000a  \000d\000a "; SQL statement:
CREATE TABLE projects 
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner VARCHAR(50),
  project_description VARCHAR(200),
  status VARCHAR(20)
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  
  [42000-214]
2022-09-03 01:12:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE projects(\000d\000a  project_id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  project_name VARCHAR(100),\000d\000a  project_owner VARCHAR(50),\000d\000a  project_description VARCHAR(200),\000d\000a  status VARCHAR(20)\000d\000a  [*]starting_date DATE,\000d\000a  finishing_date DATE,\000d\000a  date_added TIMESTAMP\000d\000a  )"; expected "ARRAY, INVISIBLE, VISIBLE, NOT NULL, NULL, AS, DEFAULT, GENERATED, ON UPDATE, NOT NULL, NULL, AUTO_INCREMENT, DEFAULT ON NULL, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY KEY, UNIQUE, NOT NULL, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner VARCHAR(50),
  project_description VARCHAR(200),
  status VARCHAR(20)
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [42001-214]
2022-09-03 01:13:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE projects(\000d\000a  project_id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  project_name VARCHAR(100),\000d\000a  project_owner VARCHAR(50),\000d\000a  project_description VARCHAR(200),\000d\000a  status VARCHAR(20),\000d\000a  project_progress INT[*](100)\000d\000a  starting_date DATE,\000d\000a  finishing_date DATE,\000d\000a  date_added TIMESTAMP\000d\000a  )"; expected "ARRAY, INVISIBLE, VISIBLE, NOT NULL, NULL, AS, DEFAULT, GENERATED, ON UPDATE, NOT NULL, NULL, AUTO_INCREMENT, DEFAULT ON NULL, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY KEY, UNIQUE, NOT NULL, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner VARCHAR(50),
  project_description VARCHAR(200),
  status VARCHAR(20),
  project_progress INT(100)
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [42001-214]
2022-09-03 01:13:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE projects(\000d\000a  project_id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  project_name VARCHAR(100),\000d\000a  project_owner VARCHAR(50),\000d\000a  project_description VARCHAR(200),\000d\000a  status VARCHAR(20),\000d\000a  project_progress INT[*](100),\000d\000a  starting_date DATE,\000d\000a  finishing_date DATE,\000d\000a  date_added TIMESTAMP\000d\000a  )"; expected "ARRAY, INVISIBLE, VISIBLE, NOT NULL, NULL, AS, DEFAULT, GENERATED, ON UPDATE, NOT NULL, NULL, AUTO_INCREMENT, DEFAULT ON NULL, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY KEY, UNIQUE, NOT NULL, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner VARCHAR(50),
  project_description VARCHAR(200),
  status VARCHAR(20),
  project_progress INT(100),
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [42001-214]
2022-09-03 01:13:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE projects(\000d\000a  project_id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  project_name VARCHAR(100),\000d\000a  project_owner VARCHAR(50),\000d\000a  project_description VARCHAR(200),\000d\000a  status VARCHAR(20),\000d\000a  project_progress INT[*](100),\000d\000a  starting_date DATE,\000d\000a  finishing_date DATE,\000d\000a  date_added TIMESTAMP\000d\000a  )"; expected "ARRAY, INVISIBLE, VISIBLE, NOT NULL, NULL, AS, DEFAULT, GENERATED, ON UPDATE, NOT NULL, NULL, AUTO_INCREMENT, DEFAULT ON NULL, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY KEY, UNIQUE, NOT NULL, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner VARCHAR(50),
  project_description VARCHAR(200),
  status VARCHAR(20),
  project_progress INT(100),
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [42001-214]
2022-09-03 01:14:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "NOT"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name NOT NULL VARCHAR(100),
  project_owner NOT NULL VARCHAR(50),
  project_description NOT NULL VARCHAR(200),
  status NOT NULL VARCHAR(20),
  project_progress INT,
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [50004-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6249)
	at org.h2.command.Parser.readIfDataType(Parser.java:6087)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6069)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5948)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9331)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:15:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "NOT"; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100),
  project_owner  VARCHAR(50),
  project_description  VARCHAR(200),
  status NOT NULL VARCHAR(20),
  project_progress INT,
  starting_date DATE,
  finishing_date DATE,
  date_added TIMESTAMP
  ) [50004-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6249)
	at org.h2.command.Parser.readIfDataType(Parser.java:6087)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6069)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5948)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9331)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:21:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "TIMESTAMP" not found; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100) NOT NULL,
  project_owner  VARCHAR(50) NOT NULL,
  project_description  VARCHAR(200) NOT NULL,
  status  VARCHAR(20) NOT NULL,
  project_progress INT NOT NULL,
  starting_date DATE NOT NULL,
  finishing_date DATE NOT NULL,
  date_added TIMESTAMP DEFAULT TIMESTAMP
  ) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5965)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9331)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:22:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "TIMESTAMP" not found; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100) NOT NULL,
  project_owner  VARCHAR(50) NOT NULL,
  project_description  VARCHAR(200) NOT NULL,
  status  VARCHAR(20) NOT NULL,
  project_progress INT NOT NULL,
  starting_date DATE NOT NULL,
  finishing_date DATE NOT NULL,
  date_added DATE DEFAULT TIMESTAMP
  ) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5965)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9331)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:22:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Function "GETDATE" not found; SQL statement:
CREATE TABLE projects(
  project_id INT AUTO_INCREMENT PRIMARY KEY,
  project_name VARCHAR(100) NOT NULL,
  project_owner  VARCHAR(50) NOT NULL,
  project_description  VARCHAR(200) NOT NULL,
  status  VARCHAR(20) NOT NULL,
  project_progress INT NOT NULL,
  starting_date DATE NOT NULL,
  finishing_date DATE NOT NULL,
  date_added date DEFAULT GETDATE()
  ) [90022-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readFunction(Parser.java:3847)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:5211)
	at org.h2.command.Parser.readTerm(Parser.java:4901)
	at org.h2.command.Parser.readFactor(Parser.java:3398)
	at org.h2.command.Parser.readSum(Parser.java:3385)
	at org.h2.command.Parser.readConcat(Parser.java:3350)
	at org.h2.command.Parser.readCondition(Parser.java:3132)
	at org.h2.command.Parser.readExpression(Parser.java:3053)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5965)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9331)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:55:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM DEPARTMENT\000d\000a\000d\000aALTER [*]TABLE department\000d\000aALTER date_create SET DEFAULT TIMESTAMP"; SQL statement:
SELECT * FROM DEPARTMENT

ALTER TABLE department
ALTER date_create SET DEFAULT TIMESTAMP [42000-214]
2022-09-03 01:55:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "TIMESTAMP" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT TIMESTAMP [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:56:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE department\000d\000aALTER date_create SET DEFAULT [*]TO TIMESTAMP"; expected "ON NULL, INTERSECTS (, NOT, EXISTS, UNIQUE, INTERSECTS"; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT TO TIMESTAMP [42001-214]
2022-09-03 01:56:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "TIMESTAMP" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT TIMESTAMP [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:56:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Function "GET_DATE" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT GET_DATE() [90022-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readFunction(Parser.java:3847)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:5211)
	at org.h2.command.Parser.readTerm(Parser.java:4901)
	at org.h2.command.Parser.readFactor(Parser.java:3398)
	at org.h2.command.Parser.readSum(Parser.java:3385)
	at org.h2.command.Parser.readConcat(Parser.java:3350)
	at org.h2.command.Parser.readCondition(Parser.java:3132)
	at org.h2.command.Parser.readExpression(Parser.java:3053)
	at org.h2.command.Parser.parseAlterTableAlterColumnSet(Parser.java:8651)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:8573)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8491)
	at org.h2.command.Parser.parseAlter(Parser.java:7548)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:57:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "GET_DATE" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT GET_DATE [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 01:58:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "[*]DESCRIBE department"; expected "DELETE, DROP, DECLARE, DEALLOCATE"; SQL statement:
DESCRIBE department [42001-214]
2022-09-03 01:59:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Function "GETDATE" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT GETDATE() [90022-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readFunction(Parser.java:3847)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:5211)
	at org.h2.command.Parser.readTerm(Parser.java:4901)
	at org.h2.command.Parser.readFactor(Parser.java:3398)
	at org.h2.command.Parser.readSum(Parser.java:3385)
	at org.h2.command.Parser.readConcat(Parser.java:3350)
	at org.h2.command.Parser.readCondition(Parser.java:3132)
	at org.h2.command.Parser.readExpression(Parser.java:3053)
	at org.h2.command.Parser.parseAlterTableAlterColumnSet(Parser.java:8651)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:8573)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8491)
	at org.h2.command.Parser.parseAlter(Parser.java:7548)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:01:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "ALTER TABLE department\000d\000a[*]MODIFY date_create DEFAULT GETDATE()"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE department
MODIFY date_create DEFAULT GETDATE() [42001-214]
2022-09-03 02:01:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "DEFAULT"; SQL statement:
ALTER TABLE department
ALTER date_create DEFAULT GETDATE() [50004-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6249)
	at org.h2.command.Parser.readIfDataType(Parser.java:6087)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6069)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5948)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8932)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:8575)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8491)
	at org.h2.command.Parser.parseAlter(Parser.java:7548)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:01:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "DEFAULT"; SQL statement:
ALTER TABLE department
ALTER date_create DEFAULT "hello" [50004-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.Parser.readIfDataType1(Parser.java:6249)
	at org.h2.command.Parser.readIfDataType(Parser.java:6087)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:6069)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5948)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8932)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:8575)
	at org.h2.command.Parser.parseAlterTable(Parser.java:8491)
	at org.h2.command.Parser.parseAlter(Parser.java:7548)
	at org.h2.command.Parser.parsePrepared(Parser.java:747)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:01:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "hello" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT "hello" [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:02:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT id [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:02:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT id [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:02:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "NAME" not found; SQL statement:
ALTER TABLE department
ALTER date_create SET DEFAULT name [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.table.Column.setDefaultExpression(Column.java:249)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:165)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:188)
	at org.h2.server.web.WebThread.run(WebThread.java:101)
	at java.base/java.lang.Thread.run(Thread.java:833)
2022-09-03 02:10:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE department(\000d\000a  id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  name VARCHAR(150),\000d\000aB VARCHAR(255) DEFAULT 'X'\000d\000a)\000d\000a\000d\000a\000d\000a[*]drop table department"; SQL statement:
CREATE TABLE department(
  id INT AUTO_INCREMENT PRIMARY KEY,
  name VARCHAR(150),
B VARCHAR(255) DEFAULT 'X'
)


drop table department [42000-214]
2022-09-03 02:10:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE department(\000d\000a  id INT AUTO_INCREMENT PRIMARY KEY,\000d\000a  name VARCHAR(150),\000d\000a[*])"; expected "identifier"; SQL statement:
CREATE TABLE department(
  id INT AUTO_INCREMENT PRIMARY KEY,
  name VARCHAR(150),
) [42001-214]
